server/models/handlers/school.js

```diff
@@ -0,0 +1,13 @@
+// Create stub which returns an array of Worker ID's given an input of school ID or school Name
+async function inputSchool(schoolID, schoolName) {
```

Comment:
Hi Simran,
It might be better to rename this function to getWorkerIdsForSchool to avoid confusion regarding the purpose of this method. Also, I think the schoolId should be the only input as that is currently the only one set as unique in the database., i.e., there may be more than one school for a schoolName so we should avoid using that parameter to ensure undesired results are not returned.

Reply:
Thank you Amy! I have adjusted the code as per your comments!